### ISSUE TITLE
[2018.07.31] SqueezeNext: Hardware-Aware Neural Network Design

### ISSUE THREAD

**Issue by unizard** [unizard/AwesomeArxiv#210]
#SqueezeNet New Version

Institute: UC Berkeley
URL: https://arxiv.org/pdf/1803.10615.pdf
Code:  https://github.com/amirgholami/SqueezeNext 
            https://github.com/luuuyi/SqueezeNext.PyTorch (He is not the author.), pytorch
Author: http://amirgholami.org/

**Summary**
In this work, we introduce SqueezeNext, a new family of neural network architectures. SqueezeNext matches AlexNet's accuracy on the ImageNet with 112x fewer parameters, and its deeper variant exceeds VGG-19's accuracy with only 4.4 Million parameters, (31x smaller). SqueezeNext also achieves better top-5 classification accuracy with 1.3x fewer parameters as compared to MobileNet, while avoiding depthwise-separable convolutions that have poor arithmetic intensity. Using hardware simulation results for power and inference speed on an embedded system, guided us to optimize the baseline model that is 2.59x/8.26x faster and 2.25x/7.5x more energy efficient as compared to SqueezeNet/AlexNet without any accuracy degradation.

**Comment by unizard** [comment]
## Architecture

![image](https://user-images.githubusercontent.com/11758940/43439098-674a563e-94cc-11e8-858b-4c14d738634f.png)
![image](https://user-images.githubusercontent.com/11758940/43439167-dc4c0cca-94cc-11e8-9a1f-f8521799614f.png)

**Comment by unizard** [comment]
## Results
![image](https://user-images.githubusercontent.com/11758940/43439235-3f9caaa0-94cd-11e8-8bdb-a27aba184bf8.png)
![image](https://user-images.githubusercontent.com/11758940/43439236-4241d0fa-94cd-11e8-83c9-2ac2ac13aec2.png)
